lupa/tests/test.py

```diff
@@ -2272,8 +2274,8 @@ def test_posargs_kwargs(self):
 
     def test_posargs_kwargs_bad(self):
         self.assertIncorrect(self.arg2, "{5, y=6, z=7}")
-        self.assertIncorrect(self.arg2, "{5, [3]=6}")
-        self.assertIncorrect(self.arg2, "{x=5, [2]=6}")  # I guess it's ok to reject this
+        self.assertIncorrect(self.arg2, "{5, [3]=6}", error=IndexError)
+        self.assertIncorrect(self.arg2, "{x=5, [2]=6}", error=IndexError)
```

Comment:
Hmm, seems worth mentioning this in the changelog. It's an incompatible change.

Reply:
Indeed. It's listed in my first message in this PR.